docs/Developers/Writing_Native_Bears.rst

```diff
@@ -51,14 +51,15 @@ each file:
 
 .. code:: python
 
-    from coalib.bears.LocalBear import LocalBear
+    import logging
 
+    from coalib.bears.LocalBear import LocalBear
 
     class HelloWorldBear(LocalBear):
         def run(self,
                 filename,
                 file):
-            self.debug("Hello World! Checking file", filename, ".")
+            logging.debug("Hello World! Checking file {} .".format(filename))
```

Comment:
Could you change the string contents in another commit? That would be btw an ideal newcomer issue ;)
(Also: You have a redundant space between `{}` and the full stop)

Reply:
Do you mean that I could file a new newcomer issue where I would ask to replace `"Hello World! Checking file", filename, "."` instead of doing so in this PR? :)

Reply:
yes exactly :+1: